.github/workflows/cypress.yml

```diff
@@ -119,22 +119,31 @@ jobs:
           node-version: '14'
           cache: npm
 
-      - name: Cache dependencies
```

Comment:
You need to keep this as by default the node_modules are not cached. You also need to keep the `id: cache` to be able to check it in the `npm ci` step.